webpack/scenes/RedHatRepositories/components/RepositorySet.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { ListView, Icon } from 'patternfly-react';
 
 import RepositoryTypeIcon from './RepositoryTypeIcon';
+// eslint-disable-next-line import/no-named-as-default
 import RepositorySetRepositories from './RepositorySetRepositories';
```

Comment:
Why disable the rule instead of importing the named import? (`import { RepositorySetRepositories }` instead of `import RepositorySetRepositories`)  And why is this needed for upgrading the vendor package?

Reply:
Because the named import is not connected to redux.
I wanted to make it work without a refactor to the component, eventually, it should migrate to the new folder structure so you will have one export for each file.

Reply:
ah ok, makes sense